desktop/core/src/desktop/lib/botserver/views.py

```diff
@@ -243,7 +252,7 @@ def _make_unfurl_payload(request, url, id_type, doc, doc_type):
           "type": "section",
           "text": {
             "type": "mrkdwn",
-            "text": "\n*<{}|Open {} {} in Hue - SQL Editor>*".format(url, name, doc_type)
+            "text": "\n*<{url}|Open {name} {doc_type} in Hue | Dialect: {dialect} | Created by: {user}>*".format(**payload_data),
```

Comment:
| and : are not pleasant to read

`Open {name} {dialect} {doc_type} created by {user}`

?

Reply:
Fixed with a more suitable heading